changelogs/fragments/1744-case-insensitive-hostname-fqdn-matching.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - redfish_config - redfish module_utils case insesitive search for situations where the hostname/FQDN case on iLO doesn't match variable's case (https://github.com/ansible-collections/community.general/pull/1744).
```

Does this affect only one module, or also others?
```suggestion
  - "redfish_config - redfish module_utils case insensitive search for situations where the hostname/FQDN case on iLO doesn't match variable's case (https://github.com/ansible-collections/community.general/pull/1744)."
```